tdrs-backend/tdpservice/users/api/login.py

```diff
@@ -59,8 +65,11 @@ def handle_user(self, request, id_token, decoded_payload):
         user = CustomAuthentication.authenticate(
             self, username=decoded_payload["email"]
         )
-        if user is not None:
+
+        if user and user.is_active:
             self.login_user(request, user, "User Found")
+        elif user and not user.is_active:
+            raise InactiveUser(f'Error: Inactive User Attempting Login {user.username}')
```

Comment:
```suggestion
            raise InactiveUser(f'Login failed, user account is inactive: {user.username}')
```
If we are presenting this to the end user being more explicit might be helpful.